vscode/src/copilot/webview/copilot.tsx

```diff
@@ -248,10 +240,21 @@ function UserMessage(props: { content: string }) {
  * An assistant message in the chat history.
  */
 function AssistantMessage(props: { content: string }) {
+  // Highlight.js needs to be called on the code blocks after they are present in the
```

Comment:
Random side note: highlight.js and Q# highlighting should probably be part of the `Markdown` element because it would be nice to have in other places we use Markdown, like API documentation.

Not asking for changes, just thought I'd note before I forget.

Reply:
I thought about that, but you can't just have a `ref` on the `<Markdown>` element itself, as it's not a DOM node, and I didn't want to go messing with the internals of the Markdown control and have to make a bigger change (in other places it's used) just for this.